benches/src/erc1155.rs

```diff
@@ -87,7 +89,9 @@ pub async fn run_with(
         (setApprovalForAllCall::SIGNATURE, receipt!(contract.setApprovalForAll(bob_addr, true))?),
         (isApprovedForAllCall::SIGNATURE, receipt!(contract.isApprovedForAll(alice_addr, bob_addr))?),
         (safeTransferFromCall::SIGNATURE, receipt!(contract.safeTransferFrom(alice_addr, bob_addr, token_1, value_1, data.clone()))?),
-        (safeBatchTransferFromCall::SIGNATURE, receipt!(contract.safeBatchTransferFrom(alice_addr, bob_addr, ids, values, data.clone()))?)
+        (safeBatchTransferFromCall::SIGNATURE, receipt!(contract.safeBatchTransferFrom(alice_addr, bob_addr, ids.clone(), values.clone(), data))?),
+        (burnCall::SIGNATURE, receipt!(contract.burn(bob_addr, token_1, value_1))?),
+        (burnBatchCall::SIGNATURE, receipt!(contract.burnBatch(bob_addr, ids, values))?),
```

Comment:
These benches are still not included in reports, see this issue https://github.com/OpenZeppelin/rust-contracts-stylus/issues/413

Reply:
@bidzyyys I think we're good here. Now it is [fixed](https://github.com/OpenZeppelin/rust-contracts-stylus/pull/417#discussion_r1861803482)